--- a/build-win32.txt
+++ b/build-win32.txt
@@ -3,7 +3,6 @@
 cpp = 'i686-w64-mingw32-g++'
 ar = 'i686-w64-mingw32-ar'
 strip = 'i686-w64-mingw32-strip'
-exe_wrapper = 'wine'
 
 [properties]
 c_link_args = ['-static', '-static-libgcc']
--- a/build-win64.txt
+++ b/build-win64.txt
@@ -3,7 +3,6 @@
 cpp = 'x86_64-w64-mingw32-g++'
 ar = 'x86_64-w64-mingw32-ar'
 strip = 'x86_64-w64-mingw32-strip'
-exe_wrapper = 'wine'
 
 [properties]
 c_link_args = ['-static', '-static-libgcc']
--- a/package-release.sh
+++ b/package-release.sh
@@ -16,9 +16,7 @@
 fi
 
 function build_arch {
-  export WINEARCH="win$1"
-  export WINEPREFIX="$DXVK_BUILD_DIR/wine.$1"
-  
+
   cd "$DXVK_SRC_DIR"
 
   meson --cross-file "$DXVK_SRC_DIR/build-win$1.txt"  \
@@ -38,8 +36,7 @@
   cp "$DXVK_BUILD_DIR/install.$1/bin/d3d10core.dll" "$DXVK_BUILD_DIR/x$1/d3d10core.dll"
   cp "$DXVK_BUILD_DIR/install.$1/bin/d3d11.dll" "$DXVK_BUILD_DIR/x$1/d3d11.dll"
   cp "$DXVK_BUILD_DIR/install.$1/bin/dxgi.dll" "$DXVK_BUILD_DIR/x$1/dxgi.dll"
-  
-  rm -R "$DXVK_BUILD_DIR/wine.$1"
+
   rm -R "$DXVK_BUILD_DIR/build.$1"
   rm -R "$DXVK_BUILD_DIR/install.$1"
 }
